etc/scripts/timetable/import.js

```diff
@@ -193,14 +210,231 @@ var _importCourses = function(ctx, courses, callback) {
     }
 
     var course = courses.pop();
+
     log().info({'id': course.orgUnitId}, 'Importing %s', course.displayName);
-    OrgUnitImport.importOrgUnit(ctx, course.orgUnitId, course, false, function(err) {
+    var options = {
+        'deleteMissing': false,
+        'checkExternalId': false
+    };
+    OrgUnitImport.importOrgUnit(ctx, course.orgUnitId, course, options, function(err) {
```

Comment:
Why wouldn't we verify we don't have unique external ids within a tree when we are importing?